components/LocaleSwitch.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <dropdown
+    ref="menu"
+    :placement="placement"
+    :offset="offset"
+  >
+    <template
+      slot="default"
+      slot-scope="{toggleMenu}"
+    >
+      <a
+        class="locale-menu"
+        href="#"
+        @click.prevent="toggleMenu()"
+      >
+        <img
+          :alt="current.name"
+          :title="current.name"
+          :src="`/img/locale-flags/${current.code}.svg`"
+          height="26"
+        >
+      </a>
+    </template>
+    <template slot="popover">
+      <ul class="locale-menu-popover">
+        <li
+          v-for="locale in locales"
+          :key="locale.code"
+        >
+          <a
+            href="#"
+            style="display: flex; align-items: center;"
+            :class="[
+              locale.code,
+              current.code === locale.code && 'active'
+            ]"
+            @click.prevent="changeLanguage(locale.code)"
+          >
+            <img
+              :alt="locale.name"
+              :title="locale.name"
+              :src="`/img/locale-flags/${locale.code}.svg`"
+              width="20"
+            > {{ locale.name }}
+          </a>
+        </li>
+      </ul>
+    </template>
+  </dropdown>
+</template>
+
+<script>
+import Dropdown from '~/components/Dropdown'
+import find from 'lodash/find'
+
+export default {
+  components: {
+    Dropdown
+  },
+  props: {
+    placement: { type: String, default: 'bottom-start' },
+    offset: { type: [String, Number], default: '16' }
+  },
+  data() {
+    return {
+      locales: process.env.locales
+    }
+  },
+  computed: {
+    current() {
+      console.log('current', this.$i18n.locale())
```

Comment:
```suggestion
```

Reply:
I thought I removed I 🤔, have to check latest push. 😬